src/app.jsx

```diff
@@ -387,6 +389,22 @@ class Application extends React.Component {
         case 'init':
         case 'wait':
             break;
+
+        /* The following events just change the State property */
+        case 'pause':
+            this.updateContainerProperty(id, system, "State", "paused");
+            break;
+        case 'unpause':
+            this.updateContainerProperty(id, system, "State", "running");
+            break;
+        case 'stop':
+            this.updateContainerProperty(id, system, "State", "stopped");
+            break;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230705-163420-8438f4c3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)